src/lib/Behat/Helper/UtilityContext.php

```diff
@@ -12,9 +12,32 @@
 use Behat\MinkExtension\Context\MinkContext;
 use Exception;
 use WebDriver\Exception\ElementNotVisible;
+use eZ\Publish\Core\MVC\Symfony\SiteAccess\Router;
+use eZ\Publish\Core\MVC\Symfony\Routing\SimplifiedRequest;
 
 class UtilityContext extends MinkContext
 {
+    /** @var \eZ\Publish\Core\MVC\Symfony\SiteAccess */
+    private $router;
+
+    /**
+     * @injectService $router @ezpublish.siteaccess_router
+     */
+    public function __construct(Router $router)
+    {
+        $this->router = $router;
+    }
+
+    public function reverseMatchRoute($siteAccessName, $route): string
```

Comment:
```suggestion
    public function reverseMatchRoute(string $siteAccessName, string $route): string
```
Also we might consider adding PHPDoc for this function.